flixel/group/FlxSpriteGroup.hx

```diff
@@ -280,6 +281,9 @@ class FlxTypedSpriteGroup<T:FlxSprite> extends FlxSprite
 		sprite.alpha *= alpha;
 		sprite.scrollFactor.copyFrom(scrollFactor);
 		sprite.cameras = _cameras; // _cameras instead of cameras because get_cameras() will not return null
+		
+		if (clipRect != null) clipRectTransform(sprite, clipRect);
+		else sprite.clipRect = null;
```

Comment:
I guess the "merge" would be `null + child.clip` which results in `child.clip`. Was your initial comment about line 286 or both of those lines?